lighthouse/blueprints/plates.py

```diff
@@ -116,7 +116,9 @@ def find_plate_from_barcode() -> FlaskResponse:
         barcodes = request.args.getlist("barcodes[]")
         logger.debug(f"Barcodes to look for: {barcodes}")
 
-        plates = [format_plate(barcode) for barcode in barcodes]
+        include_samples = (request.args.get("include_samples") == "true")
```

Comment:
I think we should make this more generic, something similar to [this](https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_fields) perhaps?

Reply:
ok, seems a good change;  what do you propose?